core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -36,10 +36,12 @@ <h2 class="oppia-preferences-page-heading-subtext">
           <div class="oppia-editable-section oppia-editor-profile-picture-container"
                tabindex="0"
                title="{{ 'I18N_PREFERENCES_CHANGE_PICTURE' | translate }}">
-            <div class="oppia-click-to-start-editing e2e-test-photo-clickable"
+            <div tabindex="0" class="oppia-click-to-start-editing e2e-test-photo-clickable"
                  (click)="showEditProfilePictureModal()"
+                 (keydown.enter)="showEditProfilePictureModal()"
                  aria-label="Edit profile picture button">
             </div>
+            <i tabindex="0" aria-label="Edit-button" (keydown.enter)="showEditProfilePictureModal()" class="fas fa-pen oppia-editor-edit-icon oppia-editor-edit-icon-pos"></i>
```

Comment:
Why is this added -- why can't we just modify the existing icon?

Reply:
i don't think @Uglycrown we need this code anymore, code below this is also doing the same job

Reply:
@seanlip @heyimShivam,  Can we solve this by removing line no 44.

Reply:
I'm not sure, you'll need to figure that out by testing the changes and seeing if they work correctly, and making edits as needed. But you shouldn't have two i tags doing the same thing.

Reply:
@seanlip, I removed line number 44 (extra i tag) and modified the existed i tag, added aria-label to it.